src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkAndDeleteModal.tsx

```diff
@@ -67,7 +66,7 @@ export const UnlinkAndDeleteModal = ({
                 <p>
                     {i18n.t(
                         'Are you sure you want to remove the link and delete the linked event? This action permanently removes the link, linked event, and all related data.',
-                    )}
+                    ) as string}
```

Comment:
Remove the `as string`. Remember to never add `as string` after the ` i18n.t(`) function